org_status/org_status.py

```diff
@@ -81,9 +81,19 @@ def main():
     parser = get_argument_parser()
     args = parser.parse_args()
 
+    styled = (lambda l, *_: l) if args.no_color else colored
+    verbose = print if args.verbose else (lambda *_: None)
+
     for Host, org in generate_fetch_jobs(args.orgs):
         token = None
 
+        try:
+            if not Host.get_host_status():
+                print(styled(f'{Host.HostName} is currently down', 'red'))
+                continue
```

Comment:
I think we should also have a else clause with `verbose(f'{Host.HostName} is up')`